src/base/components/widgets/Login.js

```diff
@@ -1,70 +1,70 @@
-import React from 'react';
-import Details from './Details.js';
-import { getServiceUrl, getDomain } from '../../servicelocator.js';
-import { Dropdown, MenuItem, Glyphicon } from 'react-bootstrap';
+// import React from 'react';
+// import Details from './Details.js';
+// import { getServiceUrl, getDomain } from '../../servicelocator.js';
+// import { Dropdown, MenuItem, Glyphicon } from 'react-bootstrap';
 
-const domain = getDomain();
+// const domain = getDomain();
 
-const LoginButton = ({ onLogin }) => (
-  <a href="#" className="btn btn-just-icon btn-simple btn-default btn-sm btn-flat pull-right">
-    <i className="material-icons dp_big">account_circle</i>
-    Login
-  </a>
-);
+// const LoginButton = ({ onLogin }) => (
+//   <a href="#" className="btn btn-just-icon btn-simple btn-default btn-sm btn-flat pull-right">
+//     <i className="material-icons dp_big">account_circle</i>
+//     Login
+//   </a>
+// );
 
-const performLogout = () => {
-  document
-    .getElementById('loginbuttoniframe')
-    .contentWindow.postMessage('logout', getServiceUrl('login'));
-};
+// const performLogout = () => {
+//   document
+//     .getElementById('loginbuttoniframe')
+//     .contentWindow.postMessage('logout', getServiceUrl('login'));
+// };
 
-export const performLogin = () => {
-  window.open(
-    `${getServiceUrl('login')}/auth/twitter?callback=${encodeURIComponent('/buttons/callback')}`,
-    'Login',
-    'height=500,width=700',
-  );
-  // document.getElementById('loginbuttoniframe').contentWindow.postMessage('login', getServiceUrl('login'));
-};
+// export const performLogin = () => {
+//   window.open(
+//     `${getServiceUrl('login')}/auth/twitter?callback=${encodeURIComponent('/buttons/callback')}`,
+//     'Login',
+//     'height=500,width=700',
+//   );
+//   // document.getElementById('loginbuttoniframe').contentWindow.postMessage('login', getServiceUrl('login'));
+// };
 
-const Login = ({ profile, readItLater = [], onLogout, onLogin }) => (
-  <Dropdown id="dropdown-custom-1" pullRight>
-    <Dropdown.Toggle className="btn-simple btn-default btn-lg btn-flat">
-      <i className="material-icons dp_big">account_circle</i>{' '}
-      {profile.temporary ? 'Guest account' : profile.name}
-    </Dropdown.Toggle>
-    <Dropdown.Menu>
-      {
-        !!readItLater.length && (readItLater.map((o, index) =>
-          <MenuItem href={o.url} key={index}>
-            <i className="material-icons dp_small with_text">bookmark</i>{o.name}
-          </MenuItem>
-        ))
-      }
-      {!!readItLater.length && (<MenuItem divider />)}
-      <MenuItem eventKey="1" href="https://core.wrioos.com/create">
-        <i className="material-icons dp_small with_text">create</i>Create new article
-      </MenuItem>
```

Comment:
Let's it be. We still need to let guest user to create new articles

Reply:
Sure